Modules/Scripted/EditorLib/LabelSelectDialog.py

```diff
@@ -0,0 +1,94 @@
+
+import qt
+import slicer
+
+#-----------------------------------------------------------------------------
+def _map_property(objfunc, name):
```

Comment:
What about the following:

```
def _map_property(objfunc, attributename):
   """Given ``objfunc`` returning a python object, this function creates a python
   :class:`property` which setter and getter are associated with ``attributename``.
   """
   [...]
```

I guess we could add this to `slicer.util`.

@mwoehlke-kitware Since you are the original author of this function, would appreciate your feedback :)